apps/dashboard/app/views/launchers/editable_form_fields/_editable_select.html.erb

```diff
@@ -34,7 +35,7 @@
 
         <button class="btn btn-warning float-end" type="button" id="<%= remove_id %>"
           data-select-toggler="remove" data-select-id="<%= field_id %>"
-          <%= disabled ? 'disabled="true"'.html_safe : nil %> >
+          <%= disabled || last_option ? 'disabled="true"'.html_safe : nil %> >
```

Comment:
We should avoid using `html_safe`.  Here it's just text so it should be fine. Not sure why the original code used it, I must not have caught it then.